digits/templates/partials/home/pretrained_model_pane.html

```diff
@@ -0,0 +1,126 @@
+                             {[ field.name ]}
+                         </small>
+                         <i class="glyphicon"
+                            ng-class="get_icon(field.name)" style="width:14px">
+                         </i>
+                       </div>
+                     </th>
+                     <th ng-repeat="field in storage.model_output_fields | filter:show(true)">
+                       <div ng-click="change_sorting(field.name, $event)">
+                         <small>
+                             {[ field.name ]}
+                         </small>
+                         <i class="glyphicon"
+                            ng-class="get_icon(field.name)" style="width:14px">
+                         </i>
+                       </div>
+                     </th>
+                   </tr>
+                </thead>
+                {[jobs = (jc.pretrained_model_jobs | filter:search_text | sort_with_empty_at_end:this);'']}
+                {[set_jobs(jobs)]}
+                <tbody ng-if="jobs.length">
+                    <tr ng-attr-id="{[ job.id ]}"
+                        tabindex="0"
+                        ng-mousedown="mousedown($index, $event)"
+                        ng-mousemove="mousemove($index, $event)"
+                        ng-mouseup="mouseup($index, $event)"
+                        ng-click="click($index, $event)"
+                        ng-keydown="keydown($event)"
+                        ng-class="{selected:job.selected}"
+                        ng-repeat="job in jobs">
+                        <td ng-bind="job.name"></td>
+                        <td><dg-framework/></td>
+                        <td ng-bind-html="'{[ job.has_labels | precision:2 ]}'|html"></td>
+                        <td class="text-{[job.status_css]}">
+                            {[ job.status ]}
+                        </td>
+                        <td start="{[job.elapsed]}">
+                            <small title="{[ print_time_diff(job.elapsed) ]}">
+                                {[ print_time_diff_terse(job.elapsed) ]}
+                            </small>
+                        </td>
+                        <td start="{[job.submitted]}">
+                            <small ng-if="!is_today(job.submitted)"
+                                   title="{[ job.submitted * 1000 | date:'medium' ]}">
+                                {[ job.submitted * 1000 | date:'MMM d, yy' ]}
+                            </small>
+                            <small ng-if="is_today(job.submitted)"
+                                   title="{[ job.submitted * 1000 | date:'medium' ]}">
+                                {[ job.submitted * 1000 | date:'h:mm a' ]}
+                            </small>
+                        </td>
+                    </tr>
+                </tbody>
+                <tbody ng-if="jobs.length == 0">
+                    <tr>
+                        <td colspan="{[fields.length]}">
```

Comment:
This colspan needs to include the displayed optional columns.

Like
```
                        {[ colspan = (storage.pretrained_model_fields | filter:show(true)).length + (storage.model_output_fields | filter:show(true)).length;'' ]}
                        <td colspan="{[colspan]}">
```
![screen shot 2016-07-18 at 10 01 36 am](https://cloud.githubusercontent.com/assets/13259615/16923495/6fa709d6-4ccf-11e6-9219-36d71c3a8b74.png)